source/imaer-gml/src/main/java/nl/overheid/aerius/gml/GMLReaderFactory.java

```diff
@@ -169,7 +169,7 @@ private static AeriusException newValidationFailed(final List<ValidationEvent> l
     final StringBuilder errorLine = new StringBuilder();
     for (final ValidationEvent event : list) {
       errorLine.append(event.getMessage());
-      errorLine.append(' ');
+      errorLine.append('|');
```

Comment:
This is not how we want to do this: with this you have to do some parsing of the message where you want to show this to a user, and would like to avoid that.
Instead, each event should end up in its own exception, and this validation should return a list of exceptions.